### PR TITLE
[ADD] field_html: add resizer in most field_html

### PR DESCRIPTION
**Pull request by Zynton**
Odoo Enterprise PR: https://github.com/odoo/enterprise/pull/14201
Jabberwock PR: https://github.com/odoo-dev/jabberwock/pull/446

### CODE REVIEW

File: addons/web_editor/static/src/js/backend/field_html.js [hunk 1]
```diff
@@ -191,6 +198,7 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
                 res_model: this.model,
                 res_id: this.res_id,
             },
+            enableResizer: this.withResizer,
```
Comment (dmo-odoo) [inline addons/web_editor/static/src/js/backend/field_html.js:201]: Why does this use two different variable names ? Why isn't it `this.enableResizer` ?

File: addons/web_editor/static/lib/jabberwock/jabberwock.css [hunk 2]
```diff
@@ -20,6 +20,28 @@ jw-editor {
...
+
+jw-resizer::before {
+    content: "☰";
+    font-size: 6px;
+    margin-top: 1px;
+    display: inline-block;
+    transform: scale(5,1);
+    color: black;
+    vertical-align: middle;
+}
```
Comment (dmo-odoo) [inline addons/web_editor/static/lib/jabberwock/jabberwock.css:44]: This entire commit should not be present inside this PR as we disussed in Discord.

File: addons/web_editor/static/src/js/backend/field_html.js (hunk 1 above)
Comment (Zinston) [reply addons/web_editor/static/src/js/backend/field_html.js:201]: Good point.

File: addons/web_editor/static/lib/jabberwock/jabberwock.css (hunk 2 above)
Comment (Zinston) [reply addons/web_editor/static/lib/jabberwock/jabberwock.css:44]: I thought you said for once we could do it this way. Fine by me...

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**